tools/benchmark-ssr/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "benchmark-ssr"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+yew = { path = "../../packages/yew", features = ["tokio", "ssr"] }
+function_router = { path = "../../examples/function_router" }
+tokio = { version = "1.19", features = ["full"] }
+jemallocator = "0.5.0"
+average = "0.13.1"
```

Comment:
Do we really need this extra dependency to calculate runtime variance? I didn't do a thorough review, but I'm not particularly impressed. For example [returning 0 instead of nan for too small samples](https://github.com/vks/average/blob/master/src/moments/variance.rs#L81) and I can't match their implementation with the claimed wikipedia source. They only take the mean once, where wikipedia has a pre- and post-update mean in their formulas.

Reply:
This is to detect when the performance was fluctuating while the benchmark is running, which is used to see if GitHub Actions is reliable in this particular run.

Other data (min, max and mean) does not provide the full picture of fluctuation of performance during the run.

Reply:
Makes sense. I'm going to trust them to have gotten their maths right, I compared a few outputs from small test cases to wolframalpa, which seems to give the correct sample variance.